ddl/table.go

```diff
@@ -264,6 +258,20 @@ func onTruncateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ erro
 	return ver, nil
 }
 
+func checkTruncateTable(t *meta.Meta, job *model.Job) (*model.TableInfo, int64, error) {
```

Comment:
Why call it `check`?

Reply:
This is a repetitive logic. It is also used in `rollingbackTruncateTable`.

Reply:
Can change the function name.